creator/extract_configs/templates/participant_conditions_config.py

```diff
@@ -0,0 +1,31 @@
+"""
+This is an extract configuration for a Participant Conditions file.
+
+See: https://www.notion.so/d3b/21e80815ba654a5fbc25f5a17b9e67a4?v=e8e02f33f87f47ff912350224cd5ab0f
+
+Required Columns:
+    Participant ID
+    Age at Onset
+    Age at Onset Units
+    Condition Name
+    Verification Status
+
+Optional Columns:
+    Age at Abatement
+    Age at Abatement Units
+    MONDO Code
+    NCIt Code
+    ICD10 Code
+    SNOMED CT
+"""
+from kf_lib_data_ingest.common.concept_schema import CONCEPT
+from kf_lib_data_ingest.etl.extract.operations import keep_map
+
+source_data_url = "{{ download_url }}"
+
+
+operations = [
+    keep_map(in_col="Participant ID", out_col=CONCEPT.PARTICIPANT.ID),
+    keep_map(in_col="Age at Onset", out_col=CONCEPT.DIAGNOSIS.EVENT_AGE_DAYS),
```

Comment:
You've ignored the age units field.

Reply:
yeah, is there a way to combine the Age + Units field to convert to `EVENT_AGE_DAYS`?

Reply:
Yes, the library will handle the conversion automatically if you use CONCEPT.DIAGNOSIS.EVENT_AGE.UNITS and CONCEPT.DIAGNOSIS.EVENT_AGE.VALUE 

See: https://github.com/kids-first/kf-lib-data-ingest/blob/615f935d58eb603b578f326add4b75d153ae8fce/kf_lib_data_ingest/common/misc.py#L18-L31

Unit options are https://github.com/kids-first/kf-lib-data-ingest/blob/615f935d58eb603b578f326add4b75d153ae8fce/kf_lib_data_ingest/common/constants.py#L23-L27

Reply:
I think we'll need to convert from the FHIR values then? https://www.hl7.org/fhir/valueset-age-units.html#definition
Maybe we can add these extra definitions to the ingest lib units?

Reply:
The codes? Yeah, could do.